workflow/workflow.go

```diff
@@ -131,3 +132,43 @@ func (v Version) String() string {
 
 	return string(v)
 }
+
+func (w Workflow) Migrate() Workflow {
```

Comment:
We will need to remember to check whether the organization is zero telemetry or no registry before doing this, but that check belongs in the CLI

Reply:
Done